src/aosm/azext_aosm/inputs/helm_chart_input.py

```diff
@@ -136,12 +191,21 @@ def get_defaults(self) -> Dict[str, Any]:
         :raises DefaultValuesNotFoundError: If no default values were found for the Helm chart.
         """
         logger.info("Getting default values for Helm chart input")
+
+        provided_config = None
+
+        if self.default_config_path:
+            if Path(self.default_config_path).exists():
+                with open(self.default_config_path, "r") as default_values:
+                    provided_config = yaml_processor.load(default_values)
+            else:
+                raise UnclassifiedUserFault(
```

Comment:
I think you moved this code from elsewhere, but this feels like a weird error to raise. Why not `FileNotFoundError`?

Reply:
Yes, I did move it from elsewhere. I think this is just because the errors in the CLI are not fixed yet